### PR TITLE
Ms2 issue14

### LINKED ISSUE #14: Write feature tests for GameController
Test that:
- GET /games returns a list of games successfully
- GET /games/{id} returns an item by id successfully
- GET /games/{id} where ID does not exist returns a 404
- aim for 100% code coverage on GameController

### PR DESCRIPTION
**Pull request by mdigrandi**
resolves #14

### CODE REVIEW

File: resources/views/game-details.blade.php [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    <title>Game Details</title>
+</head>
+<body>
+    <h1>Game Details</h1>
+    <table>
+        <thead>
+            <th>Details</th>
+        </thead>
+        <tbody>
+        @foreach($games as $game)
```
Comment (oldskoolfan) [inline resources/views/game-details.blade.php:17]: why are you adding a game-details view that loops over a game list, just like the game-list view already does? we already have a game-info template.

Comment (mdigrandi) [reply resources/views/game-details.blade.php:17]: Is this whole view redundant now? Because the game-list accomplishes what it is asking for?

Comment (oldskoolfan) [reply resources/views/game-details.blade.php:17]: yes we now have game-list (which we've always had) that shows the list of games, and game-info which shows a single game in a more-detailed view. i don't think we need game-details.

File: app/Http/Controllers/GameController.php [hunk 2]
```diff
@@ -33,8 +34,10 @@ public function getGameById(int $id)
                 return view ('game-info', [
                     'game'=>$game,
                 ]);
+
```
Comment (oldskoolfan) [inline app/Http/Controllers/GameController.php:37]:
```suggestion
$foo = 'bar';
```

**Review by oldskoolfan — APPROVED** [review]
Looks good! nice work

**Pull request merged**